docker/solr/precreate

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# This is a copy of solr 6.6's docker precreate script with some tweaks to get
+# the settings we want
+set -e
+
+echo "Executing $0 $@"
+
+if [[ "$VERBOSE" = "yes" ]]; then
+    set -x
+fi
+
+. /opt/docker-solr/scripts/run-initdb
+
+CORE=${1:-openoni}
+CONFIG_SOURCE=${2:-'/opt/solr/server/solr/configsets/data_driven_schema_configs'}
+if [[ -z $SOLR_HOME ]]; then
+    coresdir="/opt/solr/server/solr/mycores"
+    mkdir -p $coresdir
+else
+    coresdir=$SOLR_HOME
+fi
+coredir="$coresdir/$CORE"
+if [[ ! -d $coredir ]]; then
+    cp -r $CONFIG_SOURCE/ $coredir
+    touch "$coredir/core.properties"
+    echo created "$CORE"
+else
+    echo "core $CORE already exists"
+fi
+
+# Tweak the schema; ONI's schema is very explicit per field, so the dynamic
+# configs are no good for us :(
+rm -f $coredir/conf/managed-schema
```

Comment:
Hmm, this is potentially very useful.  I'm *far* from a solr expert, so if you want this to happen it may have to be all you :-/

But I question the value of a managed schema if we still have to specify all the current fields, given that the proper approach is a code rewrite that uses the managed schema properly.